.changeset/large-buttons-ring.md

```diff
@@ -0,0 +1,6 @@
+---
+'@microsoft/atlas-site': patch
```

Comment:
```suggestion
'@microsoft/atlas-site': minor
```

Reply:
(minor is for now functionality, i.e. a new size)

Reply:
updated